powerdns_exporter.go

```diff
@@ -204,9 +247,14 @@ func (e *Exporter) scrape() []StatsEntry {
 
 func (e *Exporter) collectMetrics(ch chan<- prometheus.Metric, stats []StatsEntry) {
 	statsMap := make(map[string]float64)
+
 	for _, s := range stats {
-		statsMap[s.Name] = s.Value
+		switch item := s.Item.(type) {
+		case *StatisticItem:
```

Comment:
Don't we also need to support the additional statistic items here? Or are we not interested in their values?

Having it this way makes it kinda half-implemented imho.

Reply:
We could also pull this in: https://github.com/Jimdo/powerdns_exporter/compare/master...belminf:master

Reply:
I don't understand Engin's answer to Lucas' questions 😅. 

However, I don't have a clear picture of how to implemented it for the `StatisticCollectionItem`. `statsMap` seems to be a dictionary, so what would be the key for the `StatisticItemEntry`'s inside a `StatisticCollectionItem`? A combination of their names? something like: `response-by-qtype-A`? (Taken from [here](https://github.com/Jimdo/powerdns_exporter/pull/8/files#diff-a159cd3fadb25e4a519e3439a70f892fR392))

Reply:
[Here](https://github.com/belminf/powerdns_exporter/commit/6335b4b048643ac0ab444c694012f595d1f1c328#diff-00edf31d940d2e3bc6573969a4e964caR290)'s an implementation of the other types (now I understand a little better Engin's answer)